install/installer/requirements.txt

```diff
@@ -1,2 +1,6 @@
-gitpython==3.1.0
-PyInquirer==1.0.3
+gitpython==3.1.0 # Get info from git
```

Comment:
It's safer to also have a lock file with hashes: https://github.com/jazzband/pip-tools#using-hashes.